src/main/steps/common/task-list/controllers/CaseDetailsGetController.ts

```diff
@@ -16,11 +16,205 @@ export default class CaseDetailsGetController {
     }
 
     try {
+      const data = [
```

Comment:
Please refer PRl-3914 PR for code reference to load hearing data

Reply:
done this.